src/features/move-columns/js/column-movable.js

```diff
@@ -215,6 +215,7 @@
         $timeout(function () {
           grid.api.core.notifyDataChange( uiGridConstants.dataChange.COLUMN );
           grid.api.colMovable.raise.columnPositionChanged(originalColumn.colDef, originalPosition, newPosition);
+          angular.element(`.ui-grid-header-cell.ui-grid-col${originalColumn.uid} .ui-grid-cell-contents`).focus();
```

Comment:
I am not sure what browsers this fork needs to support but I do not think IE11 supports string templating

Reply:
We already run this through babel so it doesn't matter.  I can switch it though just to keep the set of es6 features small.

Reply:
Sure we do but this fork does not use babel. Maybe we just need to update this fork at some point to use babel or update its readme to not include support for IE. In either case that only applies to if others want to use this fork which with 2,000+ I am sure people are more likely to choose something else.